kubernetes-model/src/test/java/io/fabric8/kubernetes/api/model/ConfigMapTest.java

```diff
@@ -0,0 +1,27 @@
+package io.fabric8.kubernetes.api.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import static net.javacrumbs.jsonunit.core.Option.IGNORING_ARRAY_ORDER;
+import static net.javacrumbs.jsonunit.core.Option.IGNORING_EXTRA_FIELDS;
+import static net.javacrumbs.jsonunit.core.Option.TREATING_NULL_AS_ABSENT;
+import static net.javacrumbs.jsonunit.fluent.JsonFluentAssert.assertThatJson;
+
+public class ConfigMapTest {
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Test
```

Comment:
Little nitpick, It would be nice if you follow standard Java naming conventions as mentioned [here](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#15411) .